src/util/rax_extensions.h

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2018-2019 Redis Labs Ltd. and Contributors
+ *
+ * This file is available under the Redis Labs Source Available License Agreement
+ */
+
+#pragma once
+
+#include <stddef.h>
+#include <stdbool.h>
+#include "../../deps/rax/rax.h"
+
+// Returns true if 'b' is a subset of 'a'.
+bool raxIsSubset(rax *a, rax *b);
```

Comment:
Either swap,
`bool raxIsSubset(rax *b, rax *a);`
or rephrase comment, // Returns true if 'a' is a subset of 'b'.